source/trackers.json

```diff
@@ -703,6 +794,21 @@
         "riot.im": "element",
         "graph.facebook.com": "facebook_audience",
         "app-measurement.com": "firebase",
+        "fcm.googleapis.com": "firebaseio.com",
```

Comment:
It seems to me that the domains below are worth adding to the "firebase" tracker with the "mobile_analytics" category
"firebasedynamiclinks.googleapis.com": "firebase",
"firebasedynamiclinks-ipv4.googleapis.com": "firebase",
"firebasedynamiclinks-ipv6.googleapis.com": "firebase",
"firebaseinappmessaging.googleapis.com": "firebase"

Reply:
Firebaseio itself is a mobile app development platform. 
```json
"firebaseio.com": {
    "name": "Firebase",
     /"categoryId": 8, - category "other" generally fits
    "url": "https://firebase.google.com/",
    "companyId": "google"
},
```
On the other hand most of the domains are actually used to collect user statistics